api/api.go

```diff
@@ -806,3 +814,35 @@ func (b *BlockChainAPI) prepareBlockResponse(
 
 	return blockResponse, nil
 }
+
+func (b *BlockChainAPI) getCadenceHeight(
```

Comment:
can we move this above the unsupported section (same as some other methods here), above this line: https://github.com/onflow/flow-evm-gateway/blob/0a7be0a39879a04bb23072fb20d92bc9ede73dcf/api/api.go#L546

Reply:
Good point :+1:

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/192/commits/8c868892e873c4137b1d66ed3ff290d342c01bdc